advanced/am-pattern-3/templates/am/tm/instance-1/wso2am-pattern-3-am-tm-1-deployment.yaml

```diff
@@ -12,44 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: v1
-kind: Service
-metadata:
-  name: wso2am-pattern-3-am-tm-1-service
-  namespace : {{ .Release.Namespace }}
-spec:
-  # label keys and values that must match in order to receive traffic for this service
-  selector:
-    deployment: wso2am-pattern-3-tm
-    node: wso2am-pattern-3-tm-1
-  ports:
-  # ports that this service should serve on
-  - name: jms-tcp
-    protocol: TCP
-    port: 5672
-  - name: binary-secure
-    protocol: TCP
-    port: 9711
-  - name: binary
-    protocol: TCP
-    port: 9611
-
----
-
 apiVersion: apps/v1
-kind: StatefulSet
+kind: Deployment
 metadata:
   name: wso2am-pattern-3-am-tm-1-deployment
   namespace: {{ .Release.Namespace }}
 spec:
+  replicas: {{ .Values.wso2.deployment.am.tm.replicas }}
+  minReadySeconds: {{ .Values.wso2.deployment.am.minReadySeconds }}
```

Comment:
@Chathuru @Aaquiff @ThilinaManamgoda Do we require `minReadySeconds`? I suppose we can consider the Pod to be ready as soon as the probes are completed (see [issue](https://stackoverflow.com/questions/53239081/how-does-minreadyseconds-affect-readiness-probe)).